### DISCUSSION TITLE
Switch to a different profile for gaming

### DISCUSSION THREAD

**Discussion by koluris** [seerge/g-helper#3507] [Q&A] [answered]
Long time Legion Toolkit user, there is a useful feature to switch to a different profile (turbo, custom) when a game is detected. Their games detection list is fetched from Xbox Overlay (Win key + G) in the background. The profile would then return to the default (Silent or Balanced) when the game has quit.

Is there such a feature already available on g-helper?

**Comment by seerge** [comment, marked as answer]
@koluris 

Hello, I was answering this few times already :) 

G-Helper does not monitor what happens with the system in realtime including launching games, etc (mainly to save resources, and keep app more simple) -  so it can't switch profiles based on that.

**Comment by koluris** [comment]
Thank you for your kind answer. Just for future reference, Legion Toolkit with this feature enabled takes up 0.0% - 0.1% CPU usage and 37,5MB of RAM.
